src/smp.rs

```diff
@@ -30,11 +35,9 @@ pub struct PerCpu {
     page_table: HypPageTable,
     umode_task: Once<RefCell<UmodeTask>>,
     online: Once<bool>,
+    stack_top_addr: SupervisorPageAddr,
```

Comment:
Could you store `HypStackRegion` here instead and add to it accessors for start/end of stack as well as getting it as a slice. @stillson could then use that in the backtrace PR.

Reply:
Well, that patch will have to use the virtual mapping not the physical one. Virtual mapping are static, which is one of the feature of the patch. I am not particularly happy about adding this field, but I needed between creation and initialization of CPUs, so that was good enough.

Regarding HypStackRegion, no, if we need the hypstack region we should access it from the HypMap, not here.